### PR TITLE
Update docs regarding the new default community runtime

### PR DESCRIPTION
**Pull request by pontusmelke**

### CODE REVIEW

File: cypher/cypher-docs/src/docs/dev/query-tuning.asciidoc [hunk 1]
```diff
@@ -70,18 +70,16 @@ Depending on whether Neo4j Enterprise Edition or Neo4j Community Edition is used
 Interpreted::
 In this runtime, the operators in the execution plan are chained together in a tree, where each non-leaf operator feeds from one or two child operators.
 The tree thus comprises nested iterators, and the records are streamed in a pipelined manner from the top iterator, which pulls from the next iterator and so on.
-
-[enterprise-edition]#Slotted#::
-This is very similar to the interpreted runtime, except that there are additional optimizations regarding the way in which the records are streamed through the iterators.
+Starting with 5.0 the community interpreted runtime has been updated to use the slotted runtime which used to be an enterprise-only feature.
```
Comment (renetapopova) [inline cypher/cypher-docs/src/docs/dev/query-tuning.asciidoc]:
```suggestion
In Neo4j 5.0, the community interpreted runtime has been updated to use the slotted runtime, which used to be an enterprise-only feature.
```

File: cypher/cypher-docs/src/docs/dev/query-tuning.asciidoc [hunk 2]
```diff
@@ -70,18 +70,16 @@ Depending on whether Neo4j Enterprise Edition or Neo4j Community Edition is used
...
+The slotted runtime works similar, except that there are additional optimizations regarding the way in which the records are streamed through the iterators.
```
Comment (renetapopova) [inline cypher/cypher-docs/src/docs/dev/query-tuning.asciidoc]:
```suggestion
The slotted runtime works similarly, except for some additional optimizations regarding how the records are streamed through the iterators.
```

**Review by renetapopova — APPROVED** [review]
Looks good and builds fine. I have a couple of editorial suggestions. Thanks, @pontusmelke!

**Comment by pontusmelke** [comment]
[Update docs about interpreted and slotted](https://trello.com/c/lenIYVQg/3583-update-docs-about-interpreted-and-slotted)

**Comment by renetapopova** [comment]
Thanks, @pontusmelke! Merging.

**Pull request merged**